pyproject.toml

```
@@ -26,6 +29,9 @@ keywords = [
 ]
 classifiers = [
     "Development Status :: 4 - Beta",
+    "Environment :: WebAssembly",
+    "Environment :: WebAssembly :: Emscripten",
+    "Environment :: WebAssembly :: WASI",
```

```suggestion
    "Environment :: WebAssembly :: Emscripten",
```

As @brettcannon reminds me, NumPy and SciPy work with Emscripten, but they don't cleanly work with WASI in a way that would merit the `WASI` support classifier.